drake/examples/Atlas/+atlasFrames/AtlasCoordinates.m

```diff
@@ -4,7 +4,8 @@
     function obj=AtlasCoordinates(r)
       typecheck(r,'TimeSteppingRigidBodyManipulator');
       nq = r.getNumPositions();
-      obj = obj@SingletonCoordinateFrame('atlasFrames.AtlasCoordinates',nq,'x',r.getStateFrame.coordinates(1:nq));
+      coords = r.getStateFrame();
+      obj = obj@SingletonCoordinateFrame('atlasFrames.AtlasCoordinates',nq,'x',coords(1:nq));
```

Comment:
actually, i could have written the coordinates method to (optionally) take in a matrix of indices.  but the real pain is that some code was accessing it as `coordinates(1:nq)` and others as e.g. `coordinates{1}`